File: TDDMicroExercises/CSharp/TurnTicketDispenser.SomeDependencies/TurnTicketClient.cs
namespace TDDMicroExercises.TurnTicketDispenser.SomeDependencies
{
    public class TurnTicketClient
    {
		// A class with the only goal of simulating a dependency on TurnTicket
		// that has impact on the refactoring.

		public TurnTicketClient()
        {
            var turnTicket1 = new TurnTicket(1);
			var turnTicket2 = new TurnTicket(2);
			var turnTicket3 = new TurnTicket(3);

			var num1 = turnTicket1.TurnNumber;
			var num2 = turnTicket2.TurnNumber;
			var num3 = turnTicket3.TurnNumber;
		}
    }
}

File: TDDMicroExercises/Java/TurnTicketDispenser/src/main/java/tddmicroexercises/turnticketdispenser/somedependencies/TurnTicketClient.java
package tddmicroexercises.turnticketdispenser.somedependencies;

import tddmicroexercises.turnticketdispenser.TurnTicket;

public class TurnTicketClient
{
    // A class with the only goal of simulating a dependency on TurnTicket
    // that has impact on the refactoring.

    public TurnTicketClient()
    {
        TurnTicket turnTicket1 = new TurnTicket(1);
        TurnTicket turnTicket2 = new TurnTicket(2);
        TurnTicket turnTicket3 = new TurnTicket(3);

        int num1 = turnTicket1.getTurnNumber();
        int num2 = turnTicket2.getTurnNumber();
        int num3 = turnTicket3.getTurnNumber();
    }
}
